Tasks/NuGetInstaller/task.json

```diff
@@ -27,7 +27,15 @@
             "label": "Path to Solution",
             "defaultValue":"**\\*.sln",
             "helpMarkDown": "The path to the Visual Studio Solution file",
-            "required":true
+            "required":"true"
+        },
+        {
+            "name": "nugetConfigPath",
+            "type": "filePath",
+            "label": "Path to NuGet.config",
+            "defaultValue":"",
+            "helpMarkDown": "To use feeds within this account, check in a NuGet.config which lists those feeds as sources and select it here. This will allow the build system to authenticate to those feeds, assuming you’ve added the build identity as a reader or contributor to the feed.\n\nWant to know more about feed permissions? [Learn more]( http://go.microsoft.com/fwlink/?LinkID=691232).",
```

Comment:
Yes, that's the initial approach from VSO POV. This could be extended to make it work with other external private NuGet feeds later if desired.